cfgov/unprocessed/css/print.less

```diff
@@ -90,35 +92,135 @@
         // These pseudo-classes are needed to guarantee specificity
         &:link,
         &:visited {
-            border: 0;
+            // Convert stand-alone (jump) links to standard inline links
+            border-top: 0;
+            display: inline;
             padding-top: unit( 5px / @base-font-size-px, em );
-            padding-bottom: unit( 5px / @base-font-size-px, em );
+            padding-bottom: 0;
         }
     }
 
+    .m-list__links .m-list_item {
+        // Space out stand-alone links by .5em
+        margin-bottom: unit( 8px / @base-font-size-px, em );
+    }
+
+    .o-info-unit-group {
+        .content-l_col-1-2,
+        .content-l_col-1-3,
+        .content-l_col-2-3 {
+            // Ensure content remains vertically aligned by rendering them as a table cell
+            display: table-cell !important;
+        }
+        .content-l_col-1 {
+            .grid_column( 12 );
+        }
+        .content-l_col-1-2 {
+            // Keep 50/50 info unit groups at two columns for print
+            .grid_column( 6 );
+        }
+        .content-l_col-1-3 {
+            // Keep 33/33/33 info unit groups at three columns for print
+            .grid_column( 4 );
+        }
+        .content-l_col-2-3 {
+            // Maintain 2/3 layouts for print
+            .grid_column( 8 );
+        }
+        .m-info-unit_image {
+            // Print info unit images
+            display: block;
+        }
+    }
+
+    // Force FCMs into multiple columns when printing
+    .o-featured-content-module {
+        display: flex;
+
+        &_text {
+            flex: 1;
+            padding-right: @fcm-visual-width + @grid_gutter-width;
+            padding-left: unit( @grid_gutter-width / @base-font-size-px, em );
+        }
+
+        &_visual {
+            height: 100%;
+            overflow: hidden;
+            padding-right: 0;
+            padding-bottom: 0;
+            padding-left: 0;
+            position: absolute;
+            top: 0;
+            right: 0;
+            width: @fcm-visual-width;
+        }
+
+        &_img {
+            max-width: none;
+            height: 100%;
+            position: absolute;
+        }
+
+        // Modifiers
+        &__right {
+            .o-featured-content-module_img {
+                right: 0;
+            }
+        }
+
+        &__center {
+            .o-featured-content-module_img {
+                left: 50%;
+
+                transform: translateX( -50% );
+
+                .lt-ie9 & {
+                    position: absolute;
+                    right: -100%;
+                    left: -100%;
+                    margin: auto;
+                }
+            }
+        }
+    }
+
+    .o-summary_content[aria-expanded="false"]:after {
```

Comment:
Should we be putting these component-specific styles in the component's stylesheet? Or is it easier to have them all centralized here in this print stylesheet. I particular wonder with this one, which wouldn't be affecting the layout at all.

Reply:
This is a good question and something I discussed with designers. From a developer perspective I prefer to have a single place where all the cf.gov-specific print stuff goes. Most of these CSS rules are overrides for our mobile styles necessitated by cf.gov's sidebar-based layouts and I like the idea of keeping the DS components as pure and layout-agnostic as possible. Printing is also a dying medium so it feels backwards to bake it into a DS.

W/r/t this particular rule I think you're right that it should be in the component's stylesheet since it's in the cf.gov repo and already has a very [similar hack](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/organisms/summary.less#L79-L87). I'll move it there.